### ISSUE TITLE
LightCurve.bin() - binned LightCurve missing meta data

### ISSUE THREAD

**Issue by orionlee** [lightkurve/lightkurve#1040]
#### Problem description

1. The binned lightcurve (from`lc.bin()`) does not carry the original meta data over.
2. Additionally, I think the binned lightcurve should also contain some basic bin information, e.g., bin size or possibly some bin statistics.

Motivational use case: I binned a lightcurve and use it to do `query_solar_system_objects()` to make  it run faster. It failed because binned lightcurve does not have RA / Dec in  its meta (the meta is empty).

#### Example

```python
import lightkurve

# lc is any regular LC downloaded from MAST
> lc_b = lc.bin(time_bin_size = 20 * u.minute)
> lc_b.meta
OrderedDict()

```

#### Environment

-  platform (e.g. Linux, OSX, Windows): Windows
-  lightkurve version (e.g. 1.0b6): 2.0.9
-  installation method (e.g. pip, conda, source): pip

**Comment by barentsen** [comment]
This issue has been closed because the missing meta data bug was fixed via PR #1041.

That PR did not make any changes to store bin parameters in the binned lc yet.  It would be tricky to store a record of every light curve operation parameter in a clean way in `lc.meta`, but I'd happily welcome proposals to do this as a separate issue if someone cares a lot about having this feature! :+1:

**Comment by orionlee** [comment]
Regarding storing `lc.bin()` operation arguments, the typical arguments are already stored as new columns (rather than in `lc.meta`) of the binned lc per dhomeier in discussing the PR https://github.com/lightkurve/lightkurve/pull/1041#discussion_r620783858

> Regarding other stats, time_bin_start and time_bin_size are already stored; others like no. of points per bin may not be so easy to get out of aggregate_downsample.

So I think the existing behavior addresses the typical cases. 

There might still be cases that some parameters are lost. But without any concrete cases / requirements, I think they can be tabled.
